config/generic-mks-monster8.cfg

```diff
@@ -0,0 +1,269 @@
+# This file contains common pin mappings for MKS Monster8
+# boards. To use this config, the firmware should be compiled for the
+# stm32f407. When running "make menuconfig", select the 48KiB
+# bootloader, and enable "USB for communication".
+
+# The "make flash" command does not work on the MKS Monster8. Instead,
+# after running "make", copy the generated "out/klipper.bin" file to a
+# file named "mks_monster8.bin" on an SD card or Udisk and then restart the
+# MKS Monster8 with that SD card or Udisk.
+
+# See docs/Config_Reference.md for a description of parameters.
+
+[stepper_x]
+step_pin: PC14
+dir_pin: PC13
+enable_pin: !PC15
+microsteps: 16
+rotation_distance: 40
+endstop_pin: !PA14  # PA13 for X-max; endstop have'!' is NO
+position_endstop: 0
+position_max: 200
+homing_speed: 100
```

Comment:
Please leave homing_speed at default, some printers may not take kindly to being asked to home at twice default.

Thanks
James